lib/helper/WebDriver.js

```diff
@@ -2133,6 +2133,7 @@ class WebDriver extends Helper {
         let selected = await forEachAsync(res, async el => el.isDisplayed());
```

Comment:
I thought the visible elements are filtered here, aren't they? `res, async el => el.isDisplayed()`

Reply:
Hi @PeterNgTr . No they are not filtered here. `el.isDisplayed()` returns a Boolean and this function is missing a filter by `true` .
I noticed that the filtering is done in [grabNumberOfVisibleElements](https://github.com/codecept-js/CodeceptJS/blob/master/lib/helper/WebDriver.js#L1403) but missing for this function.